new-log-viewer/src/components/Layout.tsx

```diff
@@ -165,15 +173,27 @@ const handleLogEventNumInputChange = (ev: React.ChangeEvent<HTMLInputElement>) =
  *
  * @return
  */
+// eslint-disable-next-line no-warning-comments
+// TODO: replace elements in this component once we integrate with a UI framework
+// eslint-disable-next-line max-lines-per-function
 const Layout = () => {
     const {
         fileName,
-        loadFile,
+        firstLogEventNumPerPage,
+        lastLogEventNumPerPage,
         numEvents,
+        numFilteredEvents,
         pageNum,
+
+        changeLogLevelFilter,
+        loadFile,
     } = useContext(StateContext);
     const {logEventNum} = useContext(UrlContext);
 
+    const [selectedLogLevels, setSelectedLogLevels] =
+        useState<number[]>(LOG_LEVEL_NAMES_LIST as number[]);
```

Comment:
Right, I believe the initial state would not be properly displayed because the "names" are not matching the "values" of the select-options.

Can you help create another list `LOG_LEVEL_VALUES_LIST` to be used here?